be/test/exprs/agg/aggregate_test.cpp

```diff
@@ -707,6 +707,62 @@ TEST_F(AggregateTest, test_maxby) {
     ASSERT_EQ(98.11, nullable_result_column->data_column()->get(0).get<double>());
 }
 
+TEST_F(AggregateTest, test_minby) {
+    const AggregateFunction* func = get_aggregate_function("min_by", TYPE_VARCHAR, TYPE_INT, false);
+    auto result_column = Int32Column::create();
+    auto aggr_state = ManagedAggrState::create(ctx, func);
+    auto int_column = Int32Column::create();
+    for (int i = 0; i < 10; i++) {
+        int_column->append(i);
+    }
+    auto varchar_column = BinaryColumn::create();
+    std::vector<Slice> strings{{"ccc"}, {"aaa"}, {"ddd"}, {"zzzz"}, {"ff"}, {"ff"}, {"ddd"}, {"ddd"}, {"ddd"}, {"ddd"}};
+    varchar_column->append_strings(strings);
+    Columns columns;
+    columns.emplace_back(int_column);
+    columns.emplace_back(varchar_column);
+    std::vector<const Column*> raw_columns;
+    raw_columns.resize(columns.size());
+    for (int i = 0; i < columns.size(); ++i) {
+        raw_columns[i] = columns[i].get();
+    }
+    func->update_batch_single_state(ctx, int_column->size(), raw_columns.data(), aggr_state->state());
+    func->finalize_to_column(ctx, aggr_state->state(), result_column.get());
```

Comment:
how about testing `merge`, `serialize_to_column` and `convert_to_serialize_format` in aggregation functions?